test/types/string/psahabu/perf/arguments.chpl

```diff
@@ -3,34 +3,64 @@ use Time;
 config const n = 100000000;
 config const timing = true;
 
-// passing
 var pass = "passing a string";
 var acc = 0;
-var tPassing: Timer;
-if timing then tPassing.start();
-for i in 1..n {
-  acc += receive(pass);
+
+test();
+
+// passing
+proc testPassing():Timer {
+  var tPassing: Timer;
+  if timing then tPassing.start();
```

Comment:
I wonder if we need the `if timing` bits in the function calls, because we already do the `if timing` check in `test()`.